webofneeds/won-owner-webapp/src/main/webapp/elm/Settings/Personas.elm

```diff
@@ -41,13 +42,39 @@ type alias Draft =
 
 type ValidationError
     = DisplayNameError String
+    | UrlError String
     | UnknownError String
 
 
+urlRegex : Regex.Regex
+urlRegex =
+    Regex.fromStringWith
+        { caseInsensitive = True
+        , multiline = False
+        }
+        "^(http[s]?:\\/\\/)?[^\\s([\"<,>]*\\.[^\\s[\",><]*$"
```

Comment:
This regex has a `(`, but no fitting `)`.

Reply:
As of now, everything that doesn't contain a prohibited character (whitespace) and contains a period seems to pass. `www.` is recognised as a valid URL, as is `.`, `...` or `.this-is-a-test`.

Reply:
alternative regex: `^(?:[a-z][a-z\d\-\+\.]*:(?:\/\/)?)?[\w\-\.]+(?:\.|@|:)[\w\-.]{2,}\b(?:[\w\-.~:/?#[\]@!$&'()*+,;=%])*$`